src/SqlPersistence/Saga/SqlSagaAttribute.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+namespace NServiceBus.Persistence.Sql
+{
+    /// <summary>
+    /// Exposes extra configuration options for storing <see cref="Saga{TSagaData}"/>.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class, Inherited = false)]
+    public sealed class SqlSagaAttribute : Attribute
+    {
+        /// <summary>
+        /// Identifies the Saga correlation property if it cannot be determined by inspecting the ConfigureHowToFindSaga method.
```

Comment:
@tmasternak Updated the message to be more specific that it overrides - a "means to identify" when you can't otherwise, and that IL inspection then doesn't even happen.